### PR TITLE
Feature/models

### PR DESCRIPTION
**Pull request by bartlomiejn**
Added:
- Command
- PocketRequest
- SlackCommandRequest
- SlackCommandResponse
- User
- UserProfile

### CODE REVIEW

File: Sources/Models/SlackCommandResponse.swift [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+struct SlackCommandResponse {
+    
+    // If you want the user to see their slash command displayed in the channel and
+    // no response, generate a response with responseType = .InChannel and no text
+    
+    let responseType: SlackCommandResponseType
+    let text: String?
+    let attachments: [String: String]? // Used for further customisation of a message
```
Comment (jtomanik) [inline Sources/Models/SlackCommandResponse.swift:15]: I'm a little bit uncertain about this one.
Attachments are rather complex (https://api.slack.com/docs/attachments) and they could and probably will be abstracted by something else in the future, so I would shield myself against that future change with typealias or protocol. On the other hand they are not needed to implement our current functionality and could be omitted altogether.

I'd love to hear what other team members think about it.

File: Sources/Models/User.swift [hunk 2]
```diff
@@ -0,0 +1,14 @@
+// See: https://api.slack.com/methods/users.list
```
Comment (jtomanik) [inline Sources/Models/User.swift:1]: I don't think we get Slack's user data at any point.

File: Sources/Models/UserProfile.swift [hunk 3]
```diff
@@ -0,0 +1,16 @@
+// See: https://api.slack.com/methods/users.list
```
Comment (jtomanik) [inline Sources/Models/UserProfile.swift:1]: Same here.

File: Sources/Models/PocketRequest.swift [hunk 4]
```diff
@@ -0,0 +1,11 @@
+// See: https://getpocket.com/developer/docs/v3/add
+
+struct PocketRequest {
+    
+    let url: String
+    let consumerKey: String
+    let accessToken: String
+    let title: String?
+    let tags: String? // Comma-separated list of tags to apply to the item
```
Comment (jtomanik) [inline Sources/Models/PocketRequest.swift:9]: I think in the model tags should be represented as an array and flattened later by the parser.

File: Sources/Models/PocketRequest.swift [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+    let tweetid: String?
```
Comment (mateusznadolski) [inline Sources/Models/PocketRequest.swift:10]: Test comment

**Comment by jtomanik** [comment]
Don't Merge!! Folder structure will change soon!

**Pull request closed without merging**